### PR TITLE
Users profile page

### LINKED ISSUE #38: Users Profile Page
Create a screen to show users profile stats

### PR DESCRIPTION
**Pull request by newtonneto**

### CODE REVIEW

File: src/components/MemberCard/index.tsx [hunk 1]
```diff
@@ -9,42 +9,67 @@ import {
...
+        const profileStats = response.data();
 
-        setProfile(response.data()!);
-        setIsLoading(false);
-        avatarRefIsValid(response.data()!.avatarRef) && getImage();
+        if (profileStats !== undefined) {
+          setProfile(profileStats);
+
+          await getImage(profileStats.avatarRef);
+        }
```
Comment (breno-alves) [inline src/components/MemberCard/index.tsx]: can we change this to profileStats be null or well defined?

File: src/screens/UserStats/index.tsx [hunk 2]
```diff
@@ -0,0 +1,336 @@
...
+    };
+  }, [isFocused]);
+
+  const checkMember = async () => {
+    try {
+      const response = (await partyRef.current.get()).data();
+
+      if (response !== undefined) {
+        setIsMember(response.members.includes(uuid));
+      }
```
Comment (breno-alves) [inline src/screens/UserStats/index.tsx]: can we change response to null and well defined?

File: src/screens/UserStats/index.tsx [hunk 3]
```diff
@@ -0,0 +1,336 @@
...
+    const getProfile = async () => {
+      try {
+        const response = await profileRef.current.get();
+        const profileStats = response.data();
+
+        if (profileStats !== undefined) {
+          setProfile(profileStats);
+
+          await getImage(profileStats.avatarRef);
+        }
```
Comment (breno-alves) [inline src/screens/UserStats/index.tsx]: same here

File: src/screens/UserStats/index.tsx [hunk 4]
```diff
@@ -0,0 +1,336 @@
...
+            text: 'Ok',
+          },
+        ],
+      );
+    }
+  };
+
+  const handleMember = async () => {
+    isMember ? handleBan() : handleInvite();
+  };
```
Comment (breno-alves) [inline src/screens/UserStats/index.tsx]: why are you using async function if you are not returning a promise?

**Review by breno-alves — CHANGES_REQUESTED** [review]

File: src/components/MemberCard/index.tsx (hunk 1 above)
Comment (newtonneto) [reply src/components/MemberCard/index.tsx]: The profileStats is already typed, undefined is a posible response of .data() method

File: src/screens/UserStats/index.tsx (hunk 2 above)
Comment (newtonneto) [reply src/screens/UserStats/index.tsx]: The profileStats is already typed, undefined is a posible response of .data() method

File: src/screens/UserStats/index.tsx (hunk 3 above)
Comment (newtonneto) [reply src/screens/UserStats/index.tsx]: The profileStats is already typed, undefined is a posible response of .data() method

File: src/screens/UserStats/index.tsx (hunk 4 above)
Comment (newtonneto) [reply src/screens/UserStats/index.tsx]: I forgotten to put await before methods

**Review by breno-alves — APPROVED** [review]

**Pull request merged**